plugins/BEdita/Core/tests/TestCase/Model/Table/ObjectsTableTest.php

```diff
@@ -540,7 +540,7 @@ public function testFindAncestor()
         $objects = $this->Objects->find('ancestor', [11])->toArray();
         static::assertNotEmpty($objects);
         $ids = Hash::extract($objects, '{n}.id');
-        static::assertEquals([12, 4, 2], $ids);
+        static::assertEquals([12, 2, 4], $ids);
```

Comment:
These test keep failing: it seems that different runs of the tests give different results. Does it make any sense? How can we fix this? @fquffio

Reply:
It should be sufficient to enforce sorting by ID:

```php
        $objects = $this->Objects->find('ancestor', [11])
            ->sortAsc($this->Objects->aliasField('id'))
            ->toArray();
```

Then:

```suggestion
        static::assertEquals([2, 4, 12], $ids);
```